### PR TITLE
Log validation and prevalidation times for blocks to files

### PR DESCRIPTION
**Pull request by cmmarslender**
Optionally log validation time and pre validation time of blocks to files in `CHIA_ROOT/log/`

### CODE REVIEW

File: internal/metrics/fullnode.go [hunk 1]
```diff
@@ -316,6 +317,15 @@ func (s *FullNodeServiceMetrics) Block(resp *types.WebsocketResponse) {
 	s.preValidationTime.Set(block.PreValidationTime)
 	s.validationTime.Set(block.ValidationTime)
 
+	if viper.GetBool("log-block-times") {
+		if err = utils.LogToFile("preValidationTime.log", fmt.Sprintf("%f", block.PreValidationTime)); err != nil {
```
Comment (Starttoaster) [inline internal/metrics/fullnode.go:321]: Do you have a strong opinion on using camel case for linux filenames? I prefer when my filenames are all lower-case, personally. Less headaches trying to remember case for everything. Not all shells do tab autocomplete when providing a file path.

File: internal/metrics/fullnode.go [hunk 2]
```diff
@@ -316,6 +317,15 @@ func (s *FullNodeServiceMetrics) Block(resp *types.WebsocketResponse) {
...
+			log.Error(err.Error())
+		}
+		if err = utils.LogToFile("validationTime.log", fmt.Sprintf("%f", block.ValidationTime)); err != nil {
```
Comment (Starttoaster) [inline internal/metrics/fullnode.go:324]: Same, didn't even notice the uppercase T at first here.

File: internal/metrics/fullnode.go (hunk 1 above)
Comment (cmmarslender) [reply internal/metrics/fullnode.go:321]: no not really, I just copied the code to the filename - but I generally agree with your preference actually

**Review by Starttoaster — APPROVED** [review]
LGTM

**Pull request merged**